tests/crypto/test_cases/test_ecdh.c

```diff
@@ -107,11 +107,12 @@ void ecdh_clear_buffers(void)
 
 __attribute__((noinline)) void unhexify_ecdh(void)
 {
-	expected_ss_len =
-		hex2bin(p_test_vector->p_expected_shared_secret,
-			strlen(p_test_vector->p_expected_shared_secret),
-			m_ecdh_expected_ss_buf,
-			strlen(p_test_vector->p_expected_shared_secret));
+	if (p_test_vector->p_expected_shared_secret != NULL) {
```

Comment:
This if statement is probably not required.

Reply:
Yes, good catch. I will remove it.